src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.consumer.wa;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.provider.service.CamundaConsumerApplication;
+
+@PactTestFor(providerName = "wa_task_management_api_reconfigure_task_by_id", port = "8991")
```

Comment:
provider name does not match Provider class. Should be wa_task_management_api_reconfigure_task_by_case_id"